workspaceThumbnail.js

```diff
@@ -190,9 +190,16 @@ var ThumbnailsBoxOverride = {
             this._dropPlaceholder.allocate_preferred_size(
                 ...this._dropPlaceholder.get_position());
 
-            Meta.later_add(Meta.LaterType.BEFORE_REDRAW, () => {
-                this._dropPlaceholder.hide();
-            });
+            if (Meta.later_add) {
+                Meta.later_add(Meta.LaterType.BEFORE_REDRAW, () => {
+                    this._dropPlaceholder.hide();
+                });
+            } else {
+                const laters = global.compositor.get_laters();
+                laters.add(Meta.LaterType.BEFORE_REDRAW, () => {
+                    this._dropPlaceholder.hide();
+                });
+            }
```

Comment:
Based on [GJS](https://gjs.guide/extensions/upgrading/gnome-shell-44.html#meta-later-add-and-meta-later-remove) [documentation](https://gjs-docs.gnome.org/meta12~12/meta.laters) we can use the new version of function:

> `Meta.later_add` and `Meta.later_remove` are now `Meta.Laters.add` and `Meta.Laters.remove`, respectively. 

IMO something like this should work too:

```suggestion
            const later_function = Meta.later_add || Meta.Laters.add;
            later_function.add(Meta.LaterType.BEFORE_REDRAW, () => {
                this._dropPlaceholder.hide();
            });
```

Reply:
There is a reason for using `global.compositor.get_laters()`, `Meta.Laters` is a class that needs an instance.